packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -108,6 +109,9 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
       disabled = isDisabled;
     }
 
+    required = true;
```

Comment:
Are these leftover from testing? `required` and `errorMessage` should be coming from the props, not set inside here.